controller-mock/pkg/kafka/producer.go

```diff
@@ -0,0 +1,88 @@
+package kafka
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/RHEcosystemAppEng/abb-dcs-poc/controller-mock/pkg/controller"
+	"github.com/google/uuid"
+)
+
+const (
+	HTTP_KAFKA_URL          = "http://my-bridge-bridge-route-abb-poc.apps.abb-cl01.appeng.rhecoeng.com/topics/%s"
```

Comment:
can you get this URL through an env?
```suggestion
	HTTP_KAFKA_URL          = os..Getenv(kafkaHttpHost) + "/topics/%s"
```

Reply:
sure , no prob :)